stack/setup/cloudflare.tf

```diff
@@ -33,3 +33,18 @@ resource "aws_acm_certificate_validation" "default" {
   certificate_arn         = aws_acm_certificate.main.arn
   validation_record_fqdns = cloudflare_record.validation.*.hostname
 }
+
+# If you came across this error:
+# Error: invalid zone setting "cname_flattening" (value: ) found - cannot be set as it is read only
+# 1. terraform state rm module.<module-name>.cloudflare_zone_settings_override.tls
+# 2. terraform apply
+# https://github.com/cloudflare/terraform-provider-cloudflare/issues/808
+resource "cloudflare_zone_settings_override" "tls" {
```

Comment:
Should we move this to the Cloudflare module? We can pass the settings as variable, with a default as you have it here. If null is passed explicitly, then we skip this. WDYT?